ios/Plugin/Plugin.swift

```diff
@@ -554,4 +554,27 @@ public class RadarPlugin: CAPPlugin, RadarDelegate {
         }
     }
 
+    @objc func setLogLevel(_ call: CAPPluginCall) {
+        DispatchQueue.main.async {
+            guard let levelStr = call.getString("level") else {
+                call.reject("level is required")
+                return
+            }
+            var level = RadarLogLevel.info
+            if (levelStr.caseInsensitiveCompare("none") == ComparisonResult.orderedSame) {
+                level = RadarLogLevel.none
+            } else if (levelStr.caseInsensitiveCompare("error") == ComparisonResult.orderedSame) {
+                level = RadarLogLevel.error
+            } else if (levelStr.caseInsensitiveCompare("warning") == ComparisonResult.orderedSame) {
+                level = RadarLogLevel.warning
+            } else if (levelStr.caseInsensitiveCompare("info") == ComparisonResult.orderedSame) {
+                level = RadarLogLevel.info
+            } else if (levelStr.caseInsensitiveCompare("debug") == ComparisonResult.orderedSame){
+                level = RadarLogLevel.debug
+            }
```

Comment:
FWIW, you also don't need the `ComparisonResult.` in each of these. You don't need the parentheses, either. It's enough to say `if levelStr.caseInsensitiveCompare("info") == .orderedSame`.

Reply:
Neat! Pushed an update.